packages/block-editor/src/components/block-tools/index.js

```diff
@@ -231,11 +231,12 @@ export default function BlockTools( {
 					name="__unstable-block-tools-after"
 					ref={ blockToolbarAfterRef }
 				/>
-				{ isZoomOutMode && (
-					<ZoomOutModeInserters
-						__unstableContentRef={ __unstableContentRef }
-					/>
-				) }
+				{ window.__experimentalEnableZoomedOutPatternsTab &&
```

Comment:
Is this how all experiments work? I don't love the window global 😊

Reply:
Yeah it's how we do it elsewhere.